test/merchant_repository_test.rb

```diff
@@ -0,0 +1,32 @@
+require_relative 'test_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.